### ISSUE TITLE
TNS classification not being updated

### ISSUE THREAD

**Issue by anaismoller** [astrolabsoftware/fink-science-portal#633]
TNS labels are not being updated in the portal

e.g. https://fink-portal.org/ZTF24aasmghi
which a SN in https://www.wis-tns.org/object/2024lua

https://fink-portal.org/ZTF24aatfxrw
https://www.wis-tns.org/object/2024ntb

Thanks :)

Anais

**Comment by JulienPeloton** [comment]
Thanks for reporting! This is weird indeed, I will inspect.

**Comment by JulienPeloton** [comment]
Found the bug! If you use the resolver (what is done under the hood):

```python
In [1]: import pandas as pd
In [2]: import io
In [3]: import requests
In [4]: r = requests.post(
   ...:   'https://fink-portal.org/api/v1/resolver',
   ...:   json={
   ...:     'resolver': 'tns',
   ...:     'reverse': True,
   ...:     'name': 'ZTF24aasmghi'
   ...:   }
   ...: )
In [5]: pdf = pd.read_json(io.BytesIO(r.content))
In [6]: pdf
Out[6]: 
   d:redshift  d:declination  d:fullname d:internalname        d:ra d:type       key:time
0       0.081      -8.361115  SN 2024lua   ZTF24aasmghi  326.260632  SN Ia  1720641934017
1         NaN      -8.361131  AT 2024lua   ZTF24aasmghi  326.260667    nan  1719777949758
```

there are 2 entries -- but we always take the last entry... I will fix this!

**Comment by anaismoller** [comment]
Interesting bug!

I am guessing the key:time may be useful to get the latest?

Thanks!

**Comment by JulienPeloton** [comment]
Fixed in #635 :-)
